website/templates/labels/png1/conf.php

```diff
@@ -1,20 +1,38 @@
 <?php
 
```

Comment:
(minor) to help debug, add a comment with template name ?

````
// Modern :: Wallson (beta)
````

Reply:
done